index.js

```diff
@@ -12,7 +12,19 @@
 //    See the License for the specific language governing permissions and
 //    limitations under the License.
 
-const promisify = require('es6-promisify');
+const { promisify } = require('es6-promisify');
+const { values, entries } = require('./values-entries-polyfill');
+
+/**
+ * If a patch updates a nested object,
+ * it's necessary to parse it to an array
+ * @param {*} snapVal - a JavaScript value from a DataSnapshot
```

Comment:
Please, rename param in jsdoc as in function (or vice versa), just for consistency